code/game/objects/items/weapons/tools/mods/mod_types.dm

```diff
@@ -632,3 +632,26 @@
 	UPGRADE_ITEMFLAGPLUS = LOUD
 	)
 	I.prefix = "hydraulic"
+	
+// Randomizes a bunch of weapon stats on application - can be re-rolled by re-applying it, but runs the risk of destroying the toolmod
```

Comment:
But it won't be re-rolled by re-applying it. The stats are created on item creation, and are never re-rolled.

Reply:
fixed